src/pages/slash-graphql/advanced-queries.mdx

```diff
@@ -0,0 +1,139 @@
+---
+title: "Advanced Queries with GraphQL+-"
+redirect_from:
+  - /slash-graphql/advanced-queries/
+---
+
+<h1>{props.pageContext.frontmatter.title}</h1>
+
+*These are draft docs for Slash GraphQL, which is currently in beta*
+
+Slash GraphQL also supports running advanced queries with `GraphQL+-`, a query language that is unique to Dgraph. GraphQL+- should be used by advanced users who wish to make queries and mutations using existing Dgraph client libraries, either via the HTTP or gRPC endpoints. You can learn more about existing client libraries by following this [documentation](https://dgraph.io/docs/clients/).
+
+If you are getting started with Slash GraphQL, you might want to consider using our [GraphQL APIs](https://graphql.dgraph.io/doc/) instead. It will get you quickly started on the basics of using Slash GraphQL before you go into advanced topics. 
+
+Do note that Slash GraphQL does not allow you to change the schema or create new predicates via GraphQL+-. You will not be able to access the `/alter` endpoint. You can add your schema via Slash GraphQL (either UI or API) and then add your mutations according to your Schema using GraphQL+-.
+
+## Authentication and Endpoint
+
+It is important to note that all APIs must be authenticated by an API key from the settings page. This can be accessed via Slash GraphQL UI, navigate to the [settings](https://slash.dgraph.io/_/settings) tab and you will see an option to create a new API key. Make sure to keep your API key safe, it will not be accessible once you navigate away from the page. Your API key will never expire so you don't have to worry about changing your API key. 
+
+### HTTP
+You will be able to find your base URL (GraphQL Endpoint) on your [Dashboard](https://slash.dgraph.io/_/dashboard) after your deployment is up. You will have access to `/mutate`, `/query` and `/commit` endpoints that you can use as per your convenience. 
+
+For HTTP requests, you need to make sure that the correct `Content-Type` header is sent for `/mutate` and `/query` requests. For mutations, you will be sending `Content-Type: application/rdf` and for queries, you will be sending `Content-Type: application/graphql+-`.
+
+You will also need to make sure that you pass the correct Authentication Header. The Authentication Header should be passed as `X-Auth-Token: <api-key>`. This holds true for all three available endpoints. If your Authentication Header is wrong, your requests will be rejected. You can get your API key from the settings page as described above. If you lose your API key, you can create a new one from the settings page but make sure you keep this one safe. 
```

Comment:
for all available endpoints. Rather than saying "your request will be rejected", say which status code you'll get. no need to repeat how to get an API key.